ifrs17/Import/Importers.ipynb

```diff
@@ -1059,7 +1151,14 @@
                 "\n    var mainArgs = (await GetArgsFromMainAsync<PartitionByReportingNodeAndPeriod>(dataSet)) with {ImportFormat = ImportFormats.Cashflow};",
                 "\n    if(Activity.HasErrors()) return Activity.Finish();",
                 "\n",
-                "\n    var targetArgs = await GetTargetArgsAsync(mainArgs);",
+                "\n    var targetArgs = mainArgs.Scenario == null ",
```

Comment:
Why instead of getting targetArgs you don't correct the logic to get the dependetArgs:
here you just an if with the query, in line 1160 (warning) you avoid the Skip(1), in line 1169 (foreach loop) you do the concat